etc/spack/defaults/packages.yaml

```diff
@@ -14,10 +14,16 @@
 #   ~/.spack/packages.yaml
 # -------------------------------------------------------------------------
 packages:
+  ibm-mpi:
```

Comment:
you don't need to have your specific settings merged upstream, just edit `~/.spack/packages.yaml`.